### ISSUE TITLE
Tabular figures (aka monospaced numbers)

### ISSUE THREAD

**Issue by soc** [eliheuer/titillium-web-vf#27]
Hey @eliheuer,

do you have any plans in this direction?
If not, what would be the best way for me to approach this?

Thanks,
Simon

**Comment by eliheuer** [comment]
@soc Thanks! I can add them this weekend. Or, if you want to add them feel free to make a pull request. I'd be happy to help with that if you have any questions.

**Comment by soc** [comment]
@eliheuer if you do it it would be amazing! (Because me doing it would have involved lending a mac from somewhere and buying Glyphs.) If you are happy to add the tabular figures, I'd be happy to test-drive them and report back whether everything works.
Thanks!

**Comment by eliheuer** [comment]
No, problem. I need to update some things on this repo anyway. I'll start working on this today. If you have any other improvements you want, please file more issues.

**Comment by eliheuer** [comment]
@soc Also, if you have a GNU+Linux computer you can use [fontmake](https://github.com/googlefonts/fontmake) to convert glyphs sources to [UFO](http://unifiedfontobject.org/).

**Comment by soc** [comment]
@eliheuer Ah, thanks, I thought that Glyphs was required, because the Cairo repo says in its README:

> Note that the font family in a recent versions I used connecting components, font must be generated by Glyphs.

**Comment by Gue3bara** [comment]
@eliheuer I'm not sure if Fontmake would convert the connecting components properly
